rate_counter.go

```diff
@@ -114,10 +118,9 @@ func (this *StandardRateCounter) Rate1() float64 {
 }
 
 func (this *StandardRateCounter) Count() int64 {
-	this.maybeSampleCount()
 	this.lock.RLock()
 	defer this.lock.RUnlock()
-	return this.lastCount
+	return this.counter
```

Comment:
This is racy (counter is incremented independently of any mutex). Instead of locking, you should use `atomic.LoadInt64`.

Reply:
Good call.  We should make sure this is changed.